tox.ini

```diff
@@ -87,7 +88,7 @@ commands =
     figure: /bin/bash -c "mkdir -p ./figure_test_images; python -c 'import matplotlib as mpl; print(mpl.ft2font.__file__, mpl.ft2font.__freetype_version__, mpl.ft2font.__freetype_build_type__)' > ./figure_test_images/figure_version_info.txt"
     figure: /bin/bash -c "pip freeze >> ./figure_test_images/figure_version_info.txt"
     figure: /bin/bash -c "cat ./figure_test_images/figure_version_info.txt"
-    figure: {env:PYTEST_COMMAND} -m "figure" {posargs}
+    figure: {env:PYTEST_COMMAND} -m "mpl_image_compare" --mpl --remote-data=any --mpl-generate-summary --mpl-baseline-path=https://raw.githubusercontent.com/sunpy/sunpy-figure-tests/sunpy-master/figures/{envname}/ {posargs}
```

Comment:
If we ever needed a different set of figures per-branch we would have to manually change this URL.

Reply:
This is a good question, so far we have not needed this but I wonder if that will always be the case.